### PR TITLE
Bunch of fix for (un)install scripts

### PR DESCRIPTION
**Pull request by alsoGAMER**

### CODE REVIEW

File: plugins/jre.plugin/install.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ FILE=${URL##*/}
 wget --header "Cookie: oraclelicense=a" -c "$URL" -O "$FILE"
 
 if [[ -f "$FILE" ]]; then
-    rpm -ivh "$FILE"
```
Comment (danstewart) [inline plugins/jre.plugin/install.sh]: This is intentional. See #640

File: plugins/masterpdfeditor.plugin/install.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-URL=$(curl -sL "https://code-industry.net/free-pdf-editor/" | grep -o "https://.*/master-pdf-editor.*.$(uname -m).rpm" |head -n 1 |cut -d\" -f1)
+URL=$(curl -sL "http://code-industry.net/free-pdf-editor.php" | grep -o "http://.*/master-pdf-editor.*.$(uname -m).rpm" |head -n 1 |cut -d\" -f1)
```
Comment (danstewart) [inline plugins/masterpdfeditor.plugin/install.sh]: Why change to use http?

File: plugins/simplenote.plugin/install.sh [hunk 3]
```diff
@@ -4,17 +4,30 @@ CACHEDIR="/var/cache/fedy/simplenote";
 mkdir -p "$CACHEDIR"
 cd "$CACHEDIR"
 
-API_URI="https://api.github.com"
+URL="https://github.com/Automattic/simplenote-electron/$(wget https://github.com/Automattic/simplenote-electron/releases -O - | grep -Po releases/download/v[0-9.]{5}/Simplenote-linux-x64.[0-9.]{5}.tar.gz | head -n 1)"
```
Comment (danstewart) [inline plugins/simplenote.plugin/install.sh]: What's the purpose of this change?
Is there an issue with the current install script?

Comment (alsoGAMER) [reply plugins/simplenote.plugin/install.sh]: Also my script have some issue

File: plugins/masterpdfeditor.plugin/install.sh (hunk 2 above)
Comment (alsoGAMER) [reply plugins/masterpdfeditor.plugin/install.sh]: I didn't make changes for this

File: plugins/jre.plugin/install.sh (hunk 1 above)
Comment (alsoGAMER) [reply plugins/jre.plugin/install.sh]: Again, i've make no changes to this file

**Comment by danstewart** [comment]
Not sure what's happened here.
Your fork looks fine but you've got some old code in there.

The masterpdf install script you've got is from September 2017 - https://github.com/fedy/fedy/commit/b18495abf2112b4dc5307c7dda0617f6ba67d7b5#diff-8a85bd6f2680349b3fc0ce2726e8bacf

@sudoforge Any ideas?

**Comment by alsoGAMER** [comment]
Some parts of the script have been seen from my git client as different compared to those of the repository, and has pushed also the ones.

Most of the files I've modified, they were already so after you have installed fedy
